code/src/footer.css

```diff
@@ -0,0 +1,50 @@
+footer {
+    margin-top: 20px;
+    /* height: 120px; */
+    padding: 5px;
+    background-color: white;
+    display: flex;
+    flex-direction: column;
+  }
```

Comment:
Nice that you have the CSS in different files! Makes it easy to read and understand for me who read your code for the first time. And maybe for your self as well, easy to change things and find those things you want to change. Maybe I should put all my margins from last week in different files.. :)